crates/cheatnet/src/forking/state.rs

```diff
@@ -33,18 +33,18 @@ use url::Url;
 #[derive(Debug)]
 pub struct ForkStateReader {
     client: JsonRpcClient<HttpTransport>,
-    block_id: BlockId,
+    block_number: BlockNumber,
```

Comment:
How about keeping `BlockId::BlockNumber` here? Would it be possible? I just thought that because we always convert it to BlockId

Reply:
Yes and no - `BlockId` allows other variants as well on the type level :c I will write a function that creates `BlockId` from it, it is just an enum so it is not costly